nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/assets/themes/nifi.scss

```diff
@@ -49,355 +53,100 @@ $material-primary-light-palette: (
-$nifi-canvas-accent-dark-palette: (
-    // 50 -> 900 are the PRIMARY colors defined by https://m2.material.io/design/color/the-color-system.html#tools-for-picking-colors for primary color #52bf7e
-    50: #016131,
-    100: #1A9964, // "lighter" hue for this palette.
-    200: #2cb367, //.running
-    300: #52bf7e, //.backpressure-percent
-    400: #73ca94, // color="primary" Default hue for this palette. Also .up-to-date
-    500: #9dd9b2,
-    600: #c3e8d0, //.version-control
-    700: #e6f6ec, // "darker" hue for this palette Also .bulletin-normal
-    800: #0000ff, //.endpoint, g.process-group.drop rect.border
-    900: #00ff00, //.connectable-destination, .connector.connectable
-
-    // A100 - A700 are the ANALOGOUS colors but are more customized. These colors are used to highlight, warn, denote midpoints and labelpoints, etc
-    A100: #cbaa09, //.selected
-    A200: #bda500, //.invalid, .is-missing-port, circle.context
-    A400: #f8bf47, //.backpressure-percent-warning, .bulletin-warn, .backpressure-percent.warning, text.run-status-icon
-    A700: #948b4b, //g.connection.selected rect.border, .connection-selection-path, .midpoint, .labelpoint
-
-    // These are the $nifi-canvas-accent-dark-palette PRIMARY palette contrast colors. These color do not really get defined by https://m2.material.io/design/color/the-color-system.html#tools-for-picking-colors.
-    // Instead if we look to the Angu
-    // lar Material provided palettes we see that these fields are typically rgba(black, 0.87) or white. These values are particularly important
-    // for light mode and dark mode as these values set the colors for the text when displayed against the primary background on a button, badge, chip, etc.
-    //
-    // NOTE: Care should be taken here to ensure the values meet accessibility standards.
-    //
-    // NOTE: When creating the Material palette definition mat.define-palette($nifi-canvas-accent-dark-palette, 400, 100, 700);
-    // Since 400 is the default the contrast-400 will be used as the default text color in some cases.
+$material-accent-light-palette: (
+    50: #fff8cc, 
+    100: #ebe2be, 
+    200: #d7cbb0, // lighter
+    300: #c4b5a2, 
+    400: #b09e94, 
+    500: #9c8886, // default
+    600: #8f7775, 
+    700: #836563, 
+    800: #765452, // darker
+    900: #4a3435, 
+    A100: #e7ebbe, 
+    A200: #d7d7b0, 
+    A400: #b0ab94, 
+    A700: #020202,
     contrast: (
-        50: #ffffff,
-        100: #ffffff,
-        200: #ffffff,
-        300: #ffffff,
-        400: #ffffff,
-        500: #ffffff,
-        600: rgba(black, 0.87),
-        700: rgba(black, 0.87),
-        800: rgba(black, 0.87),
-        900: rgba(black, 0.87),
-        A100: #ffffff,
-        A200: #ffffff,
-        A400: rgba(black, 0.87),
-        A700: rgba(black, 0.87),
+        50: $on-surface-dark,
+        100: $on-surface-dark,
+        200: $on-surface-dark,
+        300: $on-surface-dark,
```

Comment:
```suggestion
        300: $on-surface-light,
```

Reply:
Not enough contrast https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=C4B5A2